tests/lib/rules/jsx-no-constructed-context-values.js

```diff
@@ -31,13 +31,13 @@ const ruleTester = new RuleTester({ parserOptions });
 ruleTester.run('react-no-constructed-context-values', rule, {
   valid: parsers.all([
     {
-      code: '<Context.Provider value={props}></Context.Provider>',
+      code: 'const Component = () => <Context.Provider value={props}></Context.Provider>',
     },
     {
-      code: '<Context.Provider value={100}></Context.Provider>',
+      code: 'const Component = () => <Context.Provider value={100}></Context.Provider>',
     },
     {
-      code: '<Context.Provider value="Some string"></Context.Provider>',
+      code: 'const Component = () => <Context.Provider value="Some string"></Context.Provider>',
```

Comment:
presumably all of these existing test cases would still pass with this change?

Reply:
Yes, I change it because otherwise these test cases would pass because they are not in components, not because other any reason which is supposed to be tested.